demo/dashboard/DESCRIPTION.md

```diff
@@ -0,0 +1,3 @@
+This demo shows how you can build an interactive dashboard with gradio.
+Click on a python library on the left hand side and then on the right hand side click on the metric you'd like to see plot over time.
+Data is pulled from HuggingFace Hub datasets, e.g. [stars](https://huggingface.co/datasets/open-source-metrics/stars)
```

Comment:
Sadly there i'm not actually rendering the markdown into html, so this will literally show [stars](https://huggingface.co/datasets/open-source-metrics/stars) on the website. Maybe I should render the md

Reply:
lol, let me try that again, this will literally show 
`[stars](https://huggingface.co/datasets/open-source-metrics/stars)`

Reply:
Word - I can just put the link directly then?

Reply:
Yeah just put it for now, it won't be clickable, but I'll fix this down the line.